src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -951,45 +1054,63 @@ def generate_FDB_6_default_config(self, filename):
     device_b = json.load(
         open(os.path.join('testcases', 'testdata', 'device_b.json')))
 
+    # FSS database test harness configuration.
+    # The database File FDB_6_default_allsitedata.json has one FSS site information.
+    fss_database_config = {
+        'hostName': 'localhost',
+        'port': 8000,
+        'fileUrl': '/db_sync',
+        'filePath': os.path.join('testcases', 'testdata', 'fdb_6', 'FDB_6_default_allsitedata.json')
+    }
+
+    # GWBL database test harness configuration.
+    # The GWBL database file l_micro.zip has one GWBL W information near device_b
```

Comment:
Can you please note what the modification was?

Please also note that the condition needs to hold both for `device_a` and `device_b`.

Reply:
FDB.6     The locations for the  mentioned   GWBLs  was updated (GWBLs are moved   further than 150 km from the    respective FSS sites).